package.json

```diff
@@ -9,12 +9,12 @@
     "build": "npm run clean && mkdir dist/ && npm run babel-server && npm run babel-app",
     "clean": "rm -rf dist/",
     "coveralls": "nyc --reporter=lcov --reporter=text --require babel-register  mocha server/tests/**/*.test.js --exit",
+    "cover": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
Does this pipe the coverage to travis?

Reply:
@seyi-adeleke, it is supposed to but it currently doesn't work.